v2/charts/azure-service-operator/values.yaml

```diff
@@ -73,3 +73,14 @@ installCRDs: true
 
 # podAnnotations contain the pod annotations for Azure Service Operator
 podAnnotations: {}
+
+# multiTenant contains the value to enable multi-tenant mode for ASOv2. Also, it contains 'isCluster' flag which is
+# used to know whether to install cluster or a tenant. If multiTenant.enable is true and isCluster set to false, chart will
+# install a tenant.
+multiTenant:
+  enable: false
+#  NOTE: If we set 'isCluster' to false, tenant mode will be enabled and have to make sure we have above 'installCRDs' also set to false
+#  while tenant installation.
+  isCluster: false
```

Comment:
I didn't say this in the other PR, but I wonder if we can come up with a better name for this option... something like `clusterScopeOnly` or `webhookOnly`?

Actually now that I think about it, we already have an `azureOperatorMode`, how does this option play with that? Is it possible that we should just use that option to manage the multitenancy?

Reply:
We van just use `azureOperatorMode` option to manage multi tenancy. However, it'll not be very much clear to the end user. Where this approach just uses bools and does the job below with `azureOperatorMode`


https://github.com/Azure/azure-service-operator/pull/2460/files#diff-197e51ba2031652d2fbef03840580aaa4b78b41915813efc25a14137e53c6509R9